src/util/internals/removeFromArray.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Removes value from an array.
+ * Presence of value (and its position in an array) is determined via `Array.prototype.indexOf`
+ * @static
+ * @memberOf fabric.util
+ * @param {Array} array
+ * @param {*} value
+ * @return {Array} original array
+ */
+export const removeFromArray = (array: any[], value: any): any[] => {
```

Comment:
i m sure there is a way to tell Typescript that the returned array is the same type of the input array

Reply:
Generic types

```suggestion
export function removeFromArray<T>(array: T[], value: T): T[] {
```

Reply:
and of course T can be a mix of types. ( even if in our case often isn't )

Reply:
fat arrows aren't good for TS as functions?